libs/k6/src/common/token.js

```
@@ -0,0 +1,134 @@
+        if (extendedOptions.hasOwnProperty(key)) {
+            url.searchParams.append(key, extendedOptions[key]);
+        }
+    }
+    return fetchToken(url.toString(), extendedOptions, `enterprise iteration:${iteration})`);
+}
+
+export function getPersonalToken(tokenOptions, env='yt01') {
+    const url = new URL(`https://altinn-testtools-token-generator.azurewebsites.net/api/GetPersonalToken`);
+    let extendedOptions = addEnvAndTtlToTokenOptions(tokenOptions, env);
+    for (const key in extendedOptions) {
+        if (extendedOptions.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)